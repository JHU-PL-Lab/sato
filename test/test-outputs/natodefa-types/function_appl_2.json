[
  {
    "err_errors": [
      [
        "Error_match",
        {
          "err_match_aliases": [ "g" ],
          "err_match_val": "x + 1",
          "err_match_expected": "Function",
          "err_match_actual": "Integer"
        }
      ]
    ],
    "err_input_seq": [],
    "err_location": "g  1",
    "err_steps": 232
  }
]