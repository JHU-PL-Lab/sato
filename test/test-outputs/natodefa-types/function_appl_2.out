Type errors for:
- Input sequence : []
- Found at expr  : g  1
--------------------
* Value    : g = x + 1
* Expected : Function
* Actual   : Integer
1 errors found. No further control flows exist.
